config/routes.rb

```diff
@@ -88,7 +88,8 @@
     get '/go-fish', action: 'go_fish', as: 'go-fish'
   end
 
-  get '/nominate-a-practice', controller: 'nominate_practices', action: 'index', as: 'nominate_a_practice'
+  get '/nominate-an-innovation', controller: 'nominate_practices', action: 'index', as: 'nominate_a_practice'
```

Comment:
did we want a redirect for the old path?

Reply:
Good question! I'll ask Alex 😃

Reply:
Okay, I added some code for a redirect!